reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -70,25 +75,74 @@
+     *                  number means we send aggressively less traffic to a server
+     *                  slightly slower.
+     * @param lowQuantile the lower bound of the latency band of acceptable values.
+     *                    Any server below that value will be aggressively favored.
+     * @param highQuantile the higher bound of the latency band of acceptable values.
+     *                     Any server above that value will be aggressively penalized.
+     * @param minPendings The lower band of the average outstanding messages per server.
+     * @param maxPendings The higher band of the average outstanding messages per server.
+     * @param minAperture the minimum number of connections we want to maintain,
+     *                    independently of the load.
+     * @param maxAperture the maximum number of connections we want to maintain,
+     *                    independently of the load.
+     * @param maxRefreshPeriod the maximum time between two "refreshes" of the list of active
+     *                         ReactiveSocket. This is at that time that the slowest
+     *                         ReactiveSocket is closed.
+     * @param maxRefreshPeriodUnit Unit of the above period.
+     */
+    public LoadBalancer(
+        Publisher<List<ReactiveSocketFactory<?>>> factories,
+        double expFactor,
+        double lowQuantile,
+        double highQuantile,
+        double minPendings,
+        double maxPendings,
+        int minAperture,
+        int maxAperture,
+        long maxRefreshPeriod,
+        TimeUnit maxRefreshPeriodUnit
+    ) {
+        this.expFactor = expFactor;
+        this.lowerQuantile = new FrugalQuantile(lowQuantile);
+        this.higherQuantile = new FrugalQuantile(highQuantile);
+
+        this.activeSockets = new ArrayDeque<>();
+        this.activeFactories = new ArrayDeque<>();
         this.pendingSockets = 0;
         this.factoryRefresher = new FactoriesRefresher();
 
-        this.pendings = new Ewma(15, TimeUnit.SECONDS, (MIN_PENDINGS + MAX_PENDINGS) / 2);
-        this.targetAperture = MIN_APERTURE;
-        this.lastApertureRefresh = 0L;
-        this.refreshPeriod = Clock.unit().convert(15, TimeUnit.SECONDS);
+        this.minPendings = minPendings;
+        this.maxPendings = maxPendings;
+        this.pendings = new Ewma(15, TimeUnit.SECONDS, (minPendings + maxPendings) / 2.0);
+
+        this.minAperture = minAperture;
+        this.maxAperture = maxAperture;
+        this.targetAperture = minAperture;
+
+        this.maxRefreshPeriod = Clock.unit().convert(maxRefreshPeriod, maxRefreshPeriodUnit);
+        this.lastApertureRefresh = Clock.now();
+        this.refreshPeriod = Clock.unit().convert(15L, TimeUnit.SECONDS);
         this.lastRefresh = Clock.now();
 
         factories.subscribe(factoryRefresher);
     }
 
+    public LoadBalancer(Publisher<List<ReactiveSocketFactory<?>>> factories) {
+        this(factories,
+            4.0,
```

Comment:
Should we have these as constants so that if someone has to change one value, he can use the remaining constants?